StyleCop.Analyzers/StyleCop.Analyzers.CodeFixes/build/MSBuild15.props

```diff
@@ -0,0 +1,35 @@
+﻿<?xml version="1.0" encoding="utf-8" ?>
+<!--
+  Copyright (c) Tunnel Vision Laboratories, LLC. All Rights Reserved.
+  Licensed under the Apache License, Version 2.0. See LICENSE in the project root for license information.
+-->
+<Project xmlns="http://schemas.microsoft.com/developer/msbuild/2003">
+  <ItemGroup Condition="Exists('stylecop.json')">
+    <!--
+      If stylecop.json exists in the project directory add it to <AdditionalFiles /> automatically.  The point of
+      this is that the user sees the file in Visual Studio Solution Explorer but they don't understand that they
+      need to add it to the <AdditionalFiles /> item group.
```

Comment:
> The point of this is that the user sees the file in Visual Studio Solution Explorer but they don't understand that they need to add it to the &lt;AdditionalFiles /&gt; item group.

This text makes sense for the feature request here on GitHub, but in the implementation itself you can focus more on the 'what'. It's probably safe to leave this sentence out.